ViewPortFluidDesign/image/Layout05.svg

```diff
@@ -21,16 +21,15 @@ foreignObject {
 }
 	
 body {background: #ffe0b2; height:100%;}
-  #wrap {font-size: 4vh; color: #fff; text-align: center; text-transform: uppercase;}
-  #header-wrap {background-color: #8d6e63;}
-  #banner-wrap {background-color: #a1887f;}
-  #content-wrap {background-color: #bcaaa4;}
-  #footer-wrap {background-color: #d7ccc8;}
-
-  .header-container {width: 50%; height: 10%; background-color: #3e2723; margin: 0 auto;}
-  .banner-container {width: 50%; height: 30%; background-color: #4e342e; margin: 0 auto;}
-  .content-container {width: 50%; height: 50%; background-color: #5d4037; margin: 0 auto;}
-  .footer-container {width: 50%; height: 10%; background-color: #6d4c41; margin: 0 auto;}
+  #wrap {font-size: 4vh; height:100%; color: #fff; text-align: center; text-transform: uppercase;}
+  #header-wrap {height: 10%; background-color: #8d6e63;}
+  #banner-wrap {height: 30%; background-color: #a1887f;}
+  #content-wrap {height: 50%; background-color: #bcaaa4;}
+  #footer-wrap {height: 10%; background-color: #d7ccc8;}
+  .header-container {width: 50%; height: 100%; background-color: #3e2723; margin: 0 auto;}
+  .banner-container {width: 50%; height: 100%; background-color: #4e342e; margin: 0 auto;}
+  .content-container {width: 50%; height: 100%; background-color: #5d4037; margin: 0 auto;}
+  .footer-container {width: 50%; height: 100%; background-color: #6d4c41; margin: 0 auto;}
```

Comment:
These are each children of their respective wrap <div> elements so they can take 100% of the height given by their parent.